bindings/rust/s2n-tls-hyper/src/lib.rs

```diff
@@ -0,0 +1,5 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
```

Comment:
```suggestion
#![warn(missing_docs)]
```
We can add a lint to ensure all the public things are comments by turning on this lint.
https://doc.rust-lang.org/rustdoc/lints.html#missing_docs